File: Plugins/Aspose_Barcode_Java_for_Ruby/lib/asposebarcodejava/BarcodeImage/customizebarcodeimageresolution.rb
module Asposebarcodejava
  module CustomizeBarcodeImageResolution
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate barcode object
        bb = Rjb::import('com.aspose.barcode.BarCodeBuilder').new

        # Set the Code text for the barcode
        bb.setCodeText("1234567")

        # Set the symbology type to Code128
        bb.setSymbologyType(Rjb::import('com.aspose.barcode.Symbology').Code128)

        # Create an instance of resolution
        bb.setResolution(Rjb::import('com.aspose.barcode.Resolution').new(200,400,Rjb::import('com.aspose.barcode.ResolutionMode').Graphics))

        bb.save(data_dir + "CustomizeBarcodeImageResolution.jpg")

        # Display Status.
        puts "Customized Barcode Image Resolution, please check the output file."
    end
  end
end

File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcodeImage/BarcodeImageUtilityFeatures/CustomizeBarcodeImageResolution.php
<?php
namespace Aspose\Barcode\WorkingWithBarcodeImage\BarcodeImageUtilityFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;
use com\aspose\barcode\Resolution as Resolution;
use com\aspose\barcode\ResolutionMode as ResolutionMode;

class CustomizeBarcodeImageResolution{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $bb = new BarCodeBuilder();

        # Set the Code text for the barcode
        $bb->setCodeText("1234567");

        # Set the symbology type to Code128
        $symbology=new Symbology();
        $bb->setSymbologyType($symbology->Code128);

        # Create an instance of resolution
        $resolutionMode=new ResolutionMode();
        $bb->setResolution(new Resolution(200,400,$resolutionMode->Graphics));

        $bb->save($dataDir . "CustomizeBarcodeImageResolution.jpg");

        # Display Status.
        print "Customized Barcode Image Resolution, please check the output file.".PHP_EOL;
    }
}